code/game/mecha/mecha.dm

```diff
@@ -209,7 +209,7 @@
 /obj/mecha/proc/drop_item()//Derpfix, but may be useful in future for engineering exosuits.
 	return
 
-/obj/mecha/Hear(var/datum/speech/speech, var/rendered_message="")
+/obj/mecha/radio/Hear(var/datum/speech/speech, var/rendered_speech="")
```

Comment:
I didn't say to change it to that, and if it worked it was for utterly different reasons